tests/consensus_tests/test_hw_measurement.py

```
@@ -163,3 +163,38 @@ def assert_with_upper_bound_error(inp: int, min_value: int, upper_bound_error_pe
+    # Create collection with 50 segments and 1000 points.
+    # Each point has a payload of roughly 22B.
+    # With a request's limit of 10 points, the payload_io_read should be within 300B
+    # Before when reading payloads in every segment it would read 11 kilobytes
+    # See: <https://github.com/qdrant/qdrant/pull/6279>
+    create_collection(peer_api_uris[0], default_segment_number=50, on_disk_payload=True)
+    wait_collection_exists_and_active_on_all_peers(collection_name="test_collection", peer_api_uris=peer_api_uris)
+    upsert_random_points(peer_api_uris[0], 1000)
+    wait_collection_green(peer_api_uris[0], "test_collection")
+
+    # Send request and check payload_io_read is within limit
+    res = requests.post(
```

Ruff (0.8.2)

`requests` may be undefined, or defined from star imports

(F405)